primevue-gen/build.js

```diff
@@ -0,0 +1,29 @@
+const esbuild = require("esbuild")
+
+const myPlugin = {
+  name: "my-plugin",
+  setup(build) {
+    build.onResolve({ filter: /^vue$/ }, (args) => ({
+      path: args.path,
+      namespace: "my-plugin",
+    }))
+
+    build.onLoad({ filter: /.*/, namespace: "my-plugin" }, () => {
+      const contents = "module.exports = Vue"
+      return { contents }
+    })
```

Comment:
This is the same, but shorter and without needing to define a variable and a explicit return expression:
```
build.onLoad({ filter: /.*/, namespace: "my-plugin" }, () => ({
  contents: "module.exports = Vue"
}))
```
Moreover, it is also more consistent since it's exactly the same thing you did before in `build.onResolve`: taking advantage of the implicit return value of the arrow function.

Reply:
I see you have addressed this commit with 43381cccbaa8ee644d3c72c043c22c552fec6586.

This conversation is now resolved.